WooCommerce/WooCommerceTests/ViewRelated/CardPresentPayments/CardPresentPaymentsOnboardingUseCaseTests.swift

```diff
@@ -665,7 +665,7 @@ private extension CardPresentPaymentsOnboardingUseCaseTests {
     }
 
     enum StripePluginVersion: String {
-        case minimumSupportedVersion = "6.1.0" // Should match `CardPresentPaymentsOnboardingState` `minimumSupportedPluginVersion`
+        case minimumSupportedVersion = "6.2.0" // Should match `CardPresentPaymentsOnboardingState` `minimumSupportedPluginVersion`
         case unsupportedVersion = "5.8.1"
```

Comment:
I don't think it's a big deal, but to properly test the change, maybe the unsupported version should now be 6.1

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/6224/commits/9b03f5fd863483fd81207f037d1fc40c69ffda1f